swift-generator/pom.xml

```diff
@@ -93,5 +93,31 @@
       <groupId>io.airlift</groupId>
       <artifactId>log</artifactId>
     </dependency>
+    
+    <dependency>
+      <groupId>com.facebook.swift</groupId>
+      <artifactId>swift-test-utils</artifactId>
+      <scope>test</scope>
+    </dependency>
+    
+    <dependency>
+      <groupId>org.testng</groupId>
+      <artifactId>testng</artifactId>
+      <scope>test</scope>
+    </dependency>
+    
   </dependencies>
+  
+  <build>
+      <plugins>
+          <plugin>
```

Comment:
this `<plugin>` section doesn't actually cause anything to get generated right? lmk if I'm misreading

if we just need to set the test.output.directory property can we do that equivalently in the `<properties>` section?